README.md

```diff
@@ -108,3 +108,7 @@ options:
 
 Similar to `div`, except the next row will be appended without
 a new line being created.
+
+### cliui.clearText()
+
+Resets the text property of the current cliui.  Wrapping, width, margin, padding, border, and align set in the constructor are not reset.
```

Comment:
> reset the UI elements enqueued for rendering, maintain width and wrap properties.

(unless I'm mistaken, only `widt` and `wrap` can be set in the constructor?

Reply:
You are correct!